.gitignore

```diff
@@ -27,7 +27,6 @@ DocLib
 build.qt5
 *.sfd~
 /.project
-mscore/revision.h
```

Comment:
It might even stay here, for those that still use that hook?

Reply:
returned